set.mm

```diff
@@ -22287,24 +22287,43 @@ primitive connectives (no defined terms) on the right-hand side.
   $}
 
   ${
-    $d x y ph $.  $d y ps $.  $d y ch $.
-    mobidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
-    $( Formula-building rule for the at-most-one quantifier (deduction form).
-       (Contributed by Mario Carneiro, 7-Oct-2016.)  Remove dependency on
-       ~ ax-6 , ~ ax-7 , ~ ax-12 by adapting proof of ~ mobid .  (Revised by
-       BJ, 26-Sep-2022.) $)
-    mobidv $p |- ( ph -> ( E* x ps <-> E* x ch ) ) $=
-      ( vy weq wi wal wex wmo imbi1d albidv exbidv df-mo 3bitr4g ) ABDFGZHZDIZF
-      JCQHZDIZFJBDKCDKASUAFARTDABCQELMNBDFOCDFOP $.
+    $d y x $.  $d y ph $.  $d y ps $.
+    $( Equivalence theorem for the at-most-one quantifier.  (Contributed by BJ,
+       7-Oct-2022.) $)
+    mobi $p |- ( A. x ( ph <-> ps ) -> ( E* x ph <-> E* x ps ) ) $=
+      ( vy wb wal weq wex wmo imbi1 alimi albi syl alrimiv exbi df-mo 3bitr4g
+      wi ) ABEZCFZACDGZRZCFZDHZBUARZCFZDHZACIBCITUCUFEZDFUDUGETUHDTUBUEEZCFUHSU
+      ICABUAJKUBUECLMNUCUFDOMACDPBCDPQ $.
   $}
 
   ${
     mobii.1 $e |- ( ps <-> ch ) $.
     $( Formula-building rule for the at-most-one quantifier (inference form).
        (Contributed by NM, 9-Mar-1995.)  (Revised by Mario Carneiro,
-       17-Oct-2016.) $)
+       17-Oct-2016.)  (Proof shortened by BJ, 7-Oct-2022.) $)
     mobii $p |- ( E* x ps <-> E* x ch ) $=
-      ( wmo wb wtru a1i mobidv mptru ) ACEBCEFGABCABFGDHIJ $.
+      ( wb wmo mobi mpg ) ABEACFBCFECABCGDH $.
+  $}
+
+  ${
+    $d x ph $.
+    mobidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Formula-building rule for the at-most-one quantifier (deduction form).
+       (Contributed by Mario Carneiro, 7-Oct-2016.)  Reduce axiom dependencies
+       and shorten proof.  (Revised by BJ, 7-Oct-2022.) $)
+    mobidv $p |- ( ph -> ( E* x ps <-> E* x ch ) ) $=
+      ( wb wal wmo alrimiv mobi syl ) ABCFZDGBDHCDHFALDEIBCDJK $.
+
+    $d x y ph $.  $d y ps $.  $d y ch $.
```

Comment:
What is the reason to keep this proof?  To me it appears mostly as an expansion of mobi in the proof of mobidv.  It misses what the conventions say:
``
 Alternate proofs can also occur in other cases when an alternate proof gives some particular insight.
``

Reply:
That was a borderline case. I thought it gave that "particular insight" so I kept it as ALT.  But if you think it doesn't, then I'm going to deprecate it as OLD.

Reply:
I think OLD is correct here.

Reply:
Ok.

Reply:
Sorry @wlammen to change my mind, but I just had a second look and I think that ~mobidvALT does add insight and offers a more systematic proof, and uses the "deduction style" of proofs. It probably does not add insight *for you* and *you* can see directly that it basically is an inlining of ~mobi, as you wrote above, but this is probably not the case for someone not as versed into FOL as you, and that person would benefit from a written down ~ mobidvALT .  Therefore, I would like to keep it as ALT.

For the record:
```
MM> sh pr mobi
53           alimi.1=imbi1   $p |- ( ( ph <-> ps ) -> ( ( ph -> x = y ) <-> ( ps -> x = y ) ) )
54         syl.1=alimi     $p |- ( A. x ( ph <-> ps ) -> A. x ( ( ph -> x = y ) <-> ( ps -> x = y ) ) )
58         syl.2=albi      $p |- ( A. x ( ( ph -> x = y ) <-> ( ps -> x = y ) ) -> ( A. x ( ph -> x = y ) <-> A. x ( ps -> x = y ) ) )
59       alrimiv.1=syl   $p |- ( A. x ( ph <-> ps ) -> ( A. x ( ph -> x = y ) <-> A. x ( ps -> x = y ) ) )
60     syl.1=alrimiv   $p |- ( A. x ( ph <-> ps ) -> A. y ( A. x ( ph -> x = y ) <-> A. x ( ps -> x = y ) ) )
64     syl.2=exbi      $p |- ( A. y ( A. x ( ph -> x = y ) <-> A. x ( ps -> x = y ) ) -> ( E. y A. x ( ph -> x = y ) <-> E. y A. x ( ps -> x = y ) ) )
65   3bitr4g.1=syl   $p |- ( A. x ( ph <-> ps ) -> ( E. y A. x ( ph -> x = y ) <-> E. y A. x ( ps -> x = y ) ) )
69   3bitr4g.2=df-mo $a |- ( E* x ph <-> E. y A. x ( ph -> x = y ) )
73   3bitr4g.3=df-mo $a |- ( E* x ps <-> E. y A. x ( ps -> x = y ) )
74 mobi=3bitr4g    $p |- ( A. x ( ph <-> ps ) -> ( E* x ph <-> E* x ps ) )

MM> sh pr mobidv
17     alrimiv.1=mobidv.1 $e |- ( ph -> ( ps <-> ch ) )
18   syl.1=alrimiv      $p |- ( ph -> A. x ( ps <-> ch ) )
22   syl.2=mobi         $p |- ( A. x ( ps <-> ch ) -> ( E* x ps <-> E* x ch ) )
23 mobidv=syl         $p |- ( ph -> ( E* x ps <-> E* x ch ) )

MM> sh pr mobidvALT
36         imbid.1=mobidv.1  $e |- ( ph -> ( ps <-> ch ) )
37       albidv.1=imbi1d   $p |- ( ph -> ( ( ps -> x = y ) <-> ( ch -> x = y ) ) )
38     albidv.1=albidv   $p |- ( ph -> ( A. x ( ps -> x = y ) <-> A. x ( ch -> x = y ) ) )
39   3bitr4g.1=exbidv  $p |- ( ph -> ( E. y A. x ( ps -> x = y ) <-> E. y A. x ( ch -> x = y ) ) )
43   3bitr4g.2=df-mo   $a |- ( E* x ps <-> E. y A. x ( ps -> x = y ) )
47   3bitr4g.3=df-mo   $a |- ( E* x ch <-> E. y A. x ( ch -> x = y ) )
48 mobidvALT=3bitr4g $p |- ( ph -> ( E* x ps <-> E* x ch ) )
```

Reply:
The deduction style is exercised on various occasions from ax-1 on, and this proof adds nothing new in this respect.  I am personally not convinced.

Reply:
Ok, moved to my mathbox.